book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly define rentals as `attr_accessor` instead of `attr_reader` as you need to change it inside the `rental` class.